test/mocha.opts

```diff
@@ -0,0 +1,3 @@
+# mocha.opts
+
+  --exit
```

Comment:
Presumably this is the "quick fix" described in [mocha docs](https://mochajs.org/#-exit), can we at least add a TODO comment?

Reply:
Yep! Thank you for all these TODO recommendations! I do like me some good ol' sustainable open-source practices 👍 🎉